### PR TITLE
Adding docs with getting started guide for GCE

### PR DESCRIPTION
**Pull request by sanju-a**
# **Getting started guide for Opencontrail deployment in GCE**

This PR provides getting started guide for Opencontrail integrated with kubernetes in GCE.

### CODE REVIEW

File: docs/getting-started-guides/opencontrail-gce.md [hunk 1]
```diff
@@ -0,0 +1,289 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
```
Comment (pedro-r-marques) [inline docs/getting-started-guides/opencontrail-gce.md]: You can remove these.

File: docs/getting-started-guides/opencontrail-gce.md [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+
+Testing for upgrade is work in progress. However it will fall in the framework of using kube-push to upgrade the opencontrail components.
+
+```console
+$ KUBERNETES_PROVIDER=ubuntu ./kube-push.sh
+```
+
+Will post the details on upgrade soon.
+
+<!-- BEGIN MUNGE: GENERATED_ANALYTICS -->
```
Comment (pedro-r-marques) [inline docs/getting-started-guides/opencontrail-gce.md]: Remove the MUNGE...

**Comment by pedro-r-marques** [comment]
@sanabby Please remove the lines between BEGIN / END MUNGE at top and bottom.
I'd like to merge this ASAP.

**Comment by sanabby** [comment]
@pedro-r-marques Sorry about the delay Pedro, the notification from you on this conversation somehow was in the updates folder and I missed it. Incorporated the changes and amended the review.

**Pull request merged**